Python/Dice_Rolling-Game/Readme.md

```diff
@@ -0,0 +1,35 @@
+## Dice Rolling Simulator 📌
+
+ A python script to simulate a dice rolling game that can be played by multiple players. It provides a neat cli interface and the shows progress after each round.The dice rolls are comletely random and the first player to score 100 points wins!!</br>The random module is used in this project.
+ 
+ 
+ ## Tech Stack
+ 
+ - Python
+ 
+ 
+ ## Installing Dependencies
+ 
+ - Python 3 and above kit should work fine.
+ - No external dependencies, uses inbuilt module `random`.
+
+</br>
+
+ ## Quick Start
+ 
+ - Clone this repo in your local machine. 
+ - Start Terminal and go to the root of the local directory.
+ - Run `python dice_roll_sim.py`
```

Comment:
Hey Buddy try to add this section in these manner 

**Quick Start** (how to run in local)
<hr />
<img width="1080" alt="Screenshot 2022-03-03 at 1 01 42 PM" src="https://user-images.githubusercontent.com/78725162/158399122-5fcc235c-b2af-4607-ac48-5e97b9481f4b.png">

user require comands to run your project so mention it as shown above .